src/main/java/mate/academy/model/User.java

```diff
@@ -1,8 +1,8 @@
 package mate.academy.model;
 
 public class User {
-    private String email;
-    private String password;
+    private final String email;
+    private final String password;
```

Comment:
I can see why you did that, but usually fields in POJO classes are not final unless we want to explicitly make them final and immutable(we will talk about it more during the course), you can leave it as it is, but be careful with that next time :)